extension/_locales/es-ES/messages.json

```diff
@@ -0,0 +1,56 @@
+{
+  "extensionName": {
+    "message": "Classical Search Bar",
+  },
+  "extensionDescription": {
+    "message": "Simula la barra de búsqueda clásica agregando un botón para cambiar el motor de búsqueda.",
+  },
+  "extensionButtonTitle": {
+    "message": "Buscar por...",
+  },
+  "addButton": {
+    "message": "Añadir",
+  },
+  "resetButton": {
+    "message": "Restaurar por defecto",
```

Comment:
Should I use "Restaurar valor predeterminado", what is the different between "defecto" with "predeterminado"?